contracts/L2/gateway/L2LPTGateway.sol

```diff
@@ -25,6 +26,31 @@ contract L2LPTGateway is IL2LPTGateway, ControlledGateway, L2ArbitrumMessenger {
         l1Counterpart = _l1Counterpart;
     }
 
+    function outboundTransfer(
+        address _l1Token,
+        address _to,
+        uint256 _amount,
+        bytes calldata _data
+    ) public override whenNotPaused returns (bytes memory res) {
+        require(_l1Token == l1Lpt, "TOKEN_NOT_LPT");
+
+        (address from, bytes memory extraData) = parseOutboundData(_data);
+        require(extraData.length == 0, "CALL_HOOK_DATA_NOT_ALLOWED");
+
+        Mintable(l2Lpt).gatewayBurn(from, _amount);
```

Comment:
This needs to be changed to `burn()` - tests are failing because they currently use the LPT contract which no longer has the `gatewayBurn()` function as of the latest commit.

Reply:
Should be a quick fix - pushing something up now.

Reply:
Fixed in cbbc7b7

Reply:
Sorry I missed this. Thanks for fixing.